### PR TITLE
Docs for DPL users

### PR DESCRIPTION
**Pull request by awegrzyn**

### CODE REVIEW

**Comment by awegrzyn** [comment]
@ktf Could you verify that what I wrote makes sense?

**Comment by awegrzyn** [comment]
@ktf

File: doc/DPL.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Instructions for DPL users
+
+DPL stands for [Data Processing Layer](https://github.com/AliceO2Group/AliceO2/tree/dev/Framework/Core) and it is part of ALICE O<sup>2</sup> Framework.
+
+### Monitoring in DPL
+
+A URL to the monitoring backend can be passed through `monitoring-url` parameter. The list of supported backends is available in the [Monitoring README](https://github.com/AliceO2Group/Monitoring#monitoring-instance) file.
+
+Then, from the `O2Device` source Monitoring instance can be accessed via [Monitoring service](https://github.com/AliceO2Group/AliceO2/tree/dev/Framework/Core#monitoring-service).
```
Comment (ktf) [inline doc/DPL.md]: It's not an O2Device, it's a `DataProcessingDevice`.

**Comment by ktf** [comment]
I am away today. Will look at it tomorrow.

**Pull request merged**